src/app/api/tags/mine/route.ts

```
@@ -0,0 +1,43 @@
+    await dbConnect();
+
+    const { userId, error } = getUserId(req);
+    if (!userId) return NextResponse.json({ error }, { status: 401 });
+
+    const rawPage = parseInt(req.nextUrl?.searchParams.get('page') ?? '1', 10);
+    const rawLimit = parseInt(
+      req.nextUrl?.searchParams.get('limit') ?? '10',
+      10
+    );
+
+    if (isNaN(rawPage) || isNaN(rawLimit)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)